### ISSUE TITLE
Homepage should directly editable in the Site Editor

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#64011]
If a site is set to use a homepage and a posts page (via Reading Settings), and using the front-page.html template, then it is not possible edit the homepage in the Site Editor, unless you first choose to edit the template.

When using front-page.html, the template _is_ the content—it's one and the same. By maintaining the distinction in the Site Editor, the template vs. content complications are surfaced. 

I propose that if a site set to use a homepage and a posts page (via Reading Settings), and using a front-page.html template, then the homepage is directly editable. 

Perhaps the static page set as a homepage is "unlocked", as it's template and content in one. And the template editor is not really invoked in full here. That way you can still add a featured image, rename, trash it, etc. 

Related to https://github.com/WordPress/gutenberg/issues/63788.

**Comment by carolinan** [comment]
I think this will make it much easier for the user.

I still have concerns about users discovering that their saved changes are not available after changing themes, because they accidentally made the change in the _template_.  They should not need to worry about that.

**Comment by richtabor** [comment]
> I still have concerns about users discovering that their saved changes are not available after changing themes, because they accidentally made the change in the template. They should not need to worry about that.

Yes, that's a separate concern. I would think that perhaps available templates in the Site Editor from any theme could be available, so you can still access it.

**Comment by dhanson-wp** [comment]
Could it auto insert the Post Content block to the template? I always create a page called Home, set a Frontpage template with Post Content block, and then I edit the page content in the page instead of the site editor.

**Comment by davinder885** [comment]
> ****
![x man](https://github.com/user-attachments/assets/d2c722a6-496a-4088-a778-b57df8278b31)
